doc/source/whatsnew/v1.1.0.rst

```diff
@@ -166,6 +166,31 @@ key and type of :class:`Index`.  These now consistently raise ``KeyError`` (:iss
     ...
     KeyError: Timestamp('1970-01-01 00:00:00')
 
+GroupBy.rolling no longer returns grouped-by column in values
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*Previous behavior*:
+
+.. code-block:: ipython
+
+    In [1]: df = pd.DataFrame({"A": [1, 1, 2, 3], "B": [0, 1, 2, 3]})
```

Comment:
create the df in a separate ipython block, and show it. then just use it in the 'previous behavior' and 'current behavior' sections.